Externals.cfg

```diff
@@ -8,7 +8,7 @@ required = True
 local_path = components/cism
 protocol = git
 repo_url = https://github.com/ESCOMP/cism-wrapper
-tag = cism2_1_54
+tag = release-v2.1.00
```

Comment:
There is now a 2.1.01 tag, which changes answers relative to 2.1.00. If you have already done testing, don't worry about changing this, but I mention it in case you need to rerun testing anyway.